document/core/appendix/properties.rst

```diff
@@ -686,9 +686,9 @@ To that end, all previous typing judgements :math:`C \vdash \X{prop}` are genera
      \qquad
      (val : t_0)^\ast
      \qquad
-     S; C,\CLABELS\,(\LCATCH~[t^\ast]) \vdashinstrseq \instr^\ast : [] \to [t^\ast]
+     S; C,\CLABELS[0]:=(\LCATCH~[t^\ast]) \vdashinstrseq \instr^\ast : [] \to [t^\ast]
    }{
-     S; C,\CLABELS\,[t^\ast] \vdashadmininstr \CAUGHTadm\{\tagaddr~\val^\ast\}~\instr^\ast~\END : [] \to [t^\ast]
```

Comment:
Why change this formulation?

Reply:
This does seem misplaced. I was trying to follow the formulation for the typing of ` \DELEGATEadm` in [the formal overview](https://github.com/WebAssembly/exception-handling/blob/main/proposals/exception-handling/Exceptions-formal-overview.md#typing-rules-for-administrative-instructions), but not only this change is misplaced here, but it seems unnecessary to change this in the appendix, given the [latest comment](https://github.com/WebAssembly/exception-handling/pull/235#pullrequestreview-1181041527) on the  `DELEGATEadm` typing rule. 

Reverting this change.

Reply:
done